kotlin/build.gradle

```diff
@@ -44,7 +44,7 @@ buildscript {
   }
 
   ext.deps = [
-      'android_gradle_plugin': "com.android.tools.build:gradle:3.5.3",
+      'android_gradle_plugin': "com.android.tools.build:gradle:4.0.0-alpha09",
```

Comment:
@rjrjr This allows us to use [library desugaring](https://developer.android.com/studio/preview/features#j8-desugar) which is required to use SnakeYAML on pre-24 API level devices. I think it's because SnakeYAML is using Function types or something.